src/hooks/useGridKeyboardNavigation/useGridKeyboardNavigation.js

```diff
@@ -44,12 +44,12 @@ export default function useGridKeyboardNavigation({
   );
   const skippedInitialActiveIndexChange = useRef(false);
   const [activeIndex, setActiveIndex] = useState(isInitialActiveState ? focusItemIndexOnMount : NO_ACTIVE_INDEX);
-  const [isUsingKeyboardNav, setIsUsingKeyboardNav] = useState(focusOnMount);
+  const isUsingKeyboardNav = useRef(true);
 
   const keyboardContext = useContext(GridKeyboardNavigationContext);
 
   const onArrowNavigation = direction => {
-    setIsUsingKeyboardNav(true);
+    isUsingKeyboardNav.current = true;
```

Comment:
at first I change it to ref because I wanted to add some events and I was afraid from changing react state and renders side effects, but then I saw that there no need to add events.
I keep it as ref because it save some not needed renders but I can undo this change